eng/environments/dev.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 VER_NUM="dev-$CURR_VERSION-build.$GITHUB_SHA_SHORT"
 CONTAINER_MAIN_TAG="dev"
-VNF_OAUTH_TOKENS_ENDPOINT="https://vnf-dev.us.auth0.com/oauth/token"
+FINERACT_SECURITY_AUTH0_DOMAIN="vnf-dev.us.auth0.com"
+FINERACT_SECURITY_AUTH0_ISSUER_URI="https://vnf-dev.us.auth0.com"
+FINERACT_SECURITY_AUTH0_AUDIENCE="https://vnf-dev.us.auth0.com/userinfo" 
```

Comment:
You can build the vars using each other instead of hardcoding the hostname three times, like:
```
FINERACT_SECURITY_AUTH0_DOMAIN="vnf-dev.us.auth0.com"
FINERACT_SECURITY_AUTH0_ISSUER_URI="https://$FINERACT_SECURITY_AUTH0_DOMAIN"
FINERACT_SECURITY_AUTH0_AUDIENCE="$FINERACT_SECURITY_AUTH0_ISSUER_URI/userinfo" 
```